docs/dev/GPU-Docker-Dev.md

```diff
@@ -32,7 +36,7 @@ Now you should be able to run training jobs in this container and it will use th
 
 ```bash
 python src/levanter/main/train_lm.py \
-    --config_path config/gpt2_small.yaml
+    --config_path config/gpt2_nano.yaml
```

Comment:
small requires GCP access, nano does not, so this is a lower bar. Not sure if you want to require GCP access for this step. If so perhaps we should document how to set that up inside the contatiner before this command.

Reply:
no this is great. thanks!